js/app.js

```diff
@@ -0,0 +1,135 @@
+/*global $*/
+$(function () {
+    'use strict';
+
+    $(".store-kit").on('click', '.get', function () {
+
+        var storKit = $(this).closest(".store-kit"),
+            amount = storKit.data('price'),
+            price = $('<p class="price center-block">' + amount + '</p>'),
```

Comment:
If you're creating new elements, construct them in jQuery, not by writing HTML directly. This protects you from XSS injection problems ( a hacking method ).

Example:
`$( '<p/>', { class: 'price center-block' } )` and `$( '<a/>', { href: 'xxx', class: 'xxx' } ).text( 'Buy now' )`